LIPs/LIP-inflation-params.md

```diff
@@ -0,0 +1,96 @@
+title: Introduction of Inflation Bounds in the Inflation Adjustment Algorithm
+author: Doug Petkanics (@dob), Marco Van Dijk (@stronk)
+type: Standard Track
+status: Draft
+created: 2025-03-12
+discussions-to: https://forum.livepeer.org/t/inflation-focused-lip-discussion-thread/2753
+---
+
+## Abstract
+
+This proposal aims to enhance the current inflation adjustment mechanism of the Livepeer protocol by introducing upper and lower bounds to the inflation rate. By setting an `inflationCeiling` and `inflationFloor`, the protocol can maintain inflation within a predefined range, ensuring economic stability and predictability. In addition, this proposal suggests doubling the rate of `inflationChange` such that an equilibrium is achieved faster.
+
+## Motivation
+
+The Livepeer protocol dynamically adjusts its inflation rate each round to target a specific percentage of LPT staked (currently 50%). If participation falls below this target, inflation increases to incentivize staking; if above, it decreases. Since the migration to Arbitrum, the participation rate has hovered between 40-50%, leading to a steadily increasing inflation rate, currently at a rate exceeding 25% annually. Should inflation continue to rise, it poses risks such as:
+
+- **Perceived Over-Issuance**: Excessive token issuance can dilute value and negatively impact token perception.
+- **Economic Imbalance**: Disproportionate rewards relative to network activity can lead to inefficiencies.
+- **Market Dynamics**: Elevated inflation may exert downward pressure on token value, affecting network sustainability.
+
+While the inflation policy is theoretically pure, in that it succeeds at distributing stake in the network quickly to those who are powering the network at times of lower participation, it does face the practical above challenges. 
+
+Introducing inflation bounds will mitigate these risks by capping the inflation rate within a community-approved range.
+
+## Specification
+
+### Parameters Introduction
+
+- **`inflationCeiling`**: The maximum allowable annual inflation rate.
+- **`inflationFloor`**: The minimum allowable annual inflation rate.
+
+### Inflation Adjustment Modification
+
+The existing inflation adjustment algorithm will be modified to incorporate the following logic:
+
+1. **Calculate Target Adjustment**: Determine the inflation adjustment based on the current participation rate relative to the `targetBondingRate`. This will use the current methodology of increasing or decreasing the rate by `inflationChange`. 
+2. **Apply Bounds**: Before finalizing the adjustment:
+   - If the adjusted inflation exceeds `inflationCeiling`, set it to `inflationCeiling`.
+   - If the adjusted inflation falls below `inflationFloor`, set it to `inflationFloor`.
+
+This ensures that the inflation rate remains within the predefined bounds regardless of participation fluctuations.
+
+### Proposed Parameter Values
+
+- **`inflationCeiling`**: 750,000, corresponding to 30% per annum
```

Comment:
Suggestion: Make a note of how this maps to X% per annum i.e. link to the [denominator](https://github.com/livepeer/protocol/blob/a87d825806e9051824c212cd139786fa0f386e1d/contracts/libraries/MathUtilsV2.sol#L10) value and how to get from per round inflation to per year.